File: app-backend/src/main/java/rs/ac/singidunum/appbackend/models/ReviewModel.java
package rs.ac.singidunum.appbackend.models;

import lombok.Data;

import java.time.LocalDate;

@Data
public class ReviewModel {
    private String id;
    private ProductModel product;
    private UserModel user;
    private int score;
    private String comment;
    private LocalDate dateCreated;
}

File: BIS-project/src/app/models/reviewModel.ts
import { ProductModel } from "./productModel";
import { UserModel } from "./userModel";


export class ReviewModel{
  id?: string = "";
  product: ProductModel = null;
  user: UserModel = null;
  score: number = 0;
  comment: string = "";
  dateCreated?: Date;
}
